<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Computer Vision</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="pluginGroup">
        <property name="title">
         <string>Plugin UI</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="pluginLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>View</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QEnhancedGraphicsView" name="graphicsView"/>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="viewOriginalCheck">
             <property name="text">
              <string>View Original Image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Plugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>O&amp;ptions</string>
    </property>
    <addaction name="actionLanguage"/>
    <addaction name="actionTheme"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Plugins"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>&amp;Open Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>&amp;Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
  <action name="action_Camera">
   <property name="text">
    <string>&amp;Camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QEnhancedGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>qenhancedgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
